pytest_fixtures/api_fixtures.py

```diff
@@ -347,3 +357,9 @@ def oscap_content_path():
     local_file = f"/tmp/{file_name}"
     ssh.download_file(settings.oscap.content_path, local_file)
     return local_file
+
+
+@pytest.fixture(scope='module')
+def module_configtemplate(module_org, module_location):
```

Comment:
```suggestion
def default_pxetemplate():
```

Also, the you need not to call module_org and module_location in this fixture as you are not using them else you forgot.